clouddq/templates/dbt/macros/create_rule_binding_view.sql

```diff
@@ -107,12 +107,6 @@ all_validation_results AS (
     r.simple_rule_row_is_valid AS simple_rule_row_is_valid,
     r.complex_rule_validation_errors_count AS complex_rule_validation_errors_count,
     r.complex_rule_validation_success_flag AS complex_rule_validation_success_flag,
-    {% for ref_column_name in include_reference_columns %}
-        r.{{ ref_column_name }} AS {{ ref_column_name }},
-        {% if loop.last %}
-            {{ '\n' }}
-        {% endif %}
-    {%- endfor -%}
```

Comment:
Does this affect simple rule validation?

Reply:
no. because this is for temporary rule_binding view that we create. reference columns are not needed in here. We have added earlier as the behavior was same at rule_binding_level, but now the behavior is changed to rule_id level. This will not impact the failed records query sql.